imod/tests/test_mf6/test_mf6_simulation.py

```diff
@@ -390,9 +395,19 @@ def test_split_multiple_models_creates_expected_number_of_exchanges(
             submodel_labels, create_partition_info_mock()
         )
 
-        assert exchange_creator_mock.return_value.create_exchanges.call_count == 2
-        call1 = exchange_creator_mock.return_value.create_exchanges.call_args_list[0][0]
-        call2 = exchange_creator_mock.return_value.create_exchanges.call_args_list[1][0]
+        assert (
+            exchange_creator_mock.return_value.create_gwfgwf_exchanges.call_count == 2
+        )
+        call1 = (
+            exchange_creator_mock.return_value.create_gwfgwf_exchanges.call_args_list[
+                0
+            ][0]
+        )
+        call2 = (
+            exchange_creator_mock.return_value.create_gwfgwf_exchanges.call_args_list[
+                1
+            ][0]
+        )
```

Comment:
Argh, the linter makes these lines unreadable. I couldn't come up with a way to shorten it. I think for now you can let Ruff ignore it:

```suggestion
        assert exchange_creator_mock.return_value.create_gwfgwf_exchanges.call_count == 2  # noqa: E501
        call1 = exchange_creator_mock.return_value.create_gwfgwf_exchanges.call_args_list[0][0]  # noqa: E501
        call2 = exchange_creator_mock.return_value.create_gwfgwf_exchanges.call_args_list[1][0]  # noqa: E501

```